src/components/Book/CategoryView.tsx

```diff
@@ -7,16 +9,28 @@ type Props = {
 };
 export function BookCategoryView({ book }: Props) {
     const bestClass = book.bestInCategory ? ' best' : '';
+    const isDesktop = useMediaQuery({ query: `(min-width: 680px )` });
+    const [active, setActive] = React.useState(isDesktop ? true : false);
```

Comment:
These are not needed to be as a react state. All of these can be accomplished with SCSS.
This might be the easiest way for you but this is not correct way.

Reply:
checking for Desktop is now removed